docs/source/release-notes.html.erb

```diff
@@ -20,20 +39,21 @@ weight: 6
 
 <h2 class="govuk-heading-l" id="19-10-2021">19th October 2021</h2>
 <p class="govuk-body-m">
-   <ul class="govuk-list govuk-list--bullet">
-     <li>Add ability to withdraw an NPQ participant from a given course. <code>PUT /api/v1/participants/npq/{id}/withdraw</code></li>
-     <li>
-       <p class="govuk-body-m" >
-         Added the new endpoint to withdraw an NPQ participant from a given course.
-         <code>PUT /api/v1/participants/npq/{id}/withdraw</code>.
-       </p>
-       The previous endpoint <code>PUT /api/v1/participants/npq/{id}/withdraw</code> is deprecated and will be remove in a later version of the API.
-     </li>
-   </ul>
- </p>
+  <ul class="govuk-list govuk-list--bullet">
+    <li>Add ability to withdraw an NPQ participant from a given course. <code>PUT /api/v1/participants/npq/{id}/withdraw</code></li>
+    <li>
+      <p class="govuk-body-m" >
+        Added the new endpoint to withdraw an NPQ participant from a given course.
+        <code>PUT /api/v1/participants/npq/{id}/withdraw</code>.
+      </p>
+      The previous endpoint <code>PUT /api/v1/participants/npq/{id}/withdraw</code> is deprecated and will be remove in a later version of the API.
+    </li>
+  </ul>
+</p>
 
 <h2 class="govuk-heading-l" id="5-10-2021">5th October 2021</h2>
 <p class="govuk-body-m">Update withdrawal and deferral reason codes.</p>
+<p class="govuk-body-m">Added created_at date to NPQ applications (GET endpoints).</p>
```

Comment:
Something weird going on with the diff here, we have this addition on the 27th of October